### ISSUE TITLE
Server side rendering invalid checksum

### ISSUE THREAD

**Issue by giladl82** [daviferreira/react-sanfona#138]
When using server side rendering, I get the following warning

Warning: React attempted to reuse markup in a container but the checksum was invalid. This generally means that you are using server rendering and the markup generated on the server was not what the client was expecting. React injected new markup to compensate which works but you have lost many of the benefits of server rendering. Instead, figure out why the markup being generated is different on the client or server:
 (client) t-sanfona-item-body-0e1c15bb-31a7-441c-8
 (server) t-sanfona-item-body-e2108965-ac06-4b9a-8

I got this message even when the accordion was totally static

Thanks

**Comment by daviferreira** [comment]
@giladl82 Can you provide a fiddle/example? :)

**Comment by aelohin** [comment]
Hi, did you find a solution to this issue? I have it too. Everything works very well, once I do the changes to work the app with SSR then this issue is created.  I will try to create an online example but maybe you already know the answer.

Thanks!

**Comment by giladl82** [comment]
Hi,
I sorry to say I didn't find the solution to this problem.
I've neglected the project so I made no progress to resolve.
However, I've on next.js
It's a framework for server side rendering.
You might like to give it a try.


On Aug 17, 2018 21:52, "Alex Torres" <notifications@github.com> wrote:

Hi, did you find a solution to this issue? I have it too. Everything works
very well, once I do the changes to work the app with SSR then this issue
is created. I will try to create an online example but maybe you already
know the answer.

Thanks!

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/daviferreira/react-sanfona/issues/138#issuecomment-413957852>,
or mute the thread
